java/arcs/tools/Recipe2Plan.kt

```diff
@@ -0,0 +1,48 @@
+package arcs.tools
+
+import arcs.core.storage.api.DriverAndKeyConfigurator
+import com.github.ajalt.clikt.core.CliktCommand
+import com.github.ajalt.clikt.parameters.arguments.argument
+import com.github.ajalt.clikt.parameters.arguments.default
+import com.github.ajalt.clikt.parameters.arguments.multiple
+import com.github.ajalt.clikt.parameters.options.flag
+import com.github.ajalt.clikt.parameters.options.option
+import com.github.ajalt.clikt.parameters.types.file
+import com.squareup.kotlinpoet.FileSpec
+
+/** Generates plans from recipes. */
+class Recipe2Plan : CliktCommand(
+    help = """Generate plans from recipes.
+    
+    This script reads serialized manifests and generates Kotlin files with [Plan] classes.""",
+    printHelpOnEmptyArgs = true
+) {
+    private val outputFile by argument(help = "output Kotlin filepath, e.g. path/to/File.kt")
+        .file()
+    // TODO(b/161994250): Package should be derived from proto
+    private val packageName by argument(help = "scope to specified package").default("")
+    private val manifests by argument(
+        help = "paths to protobuf-serialized manifests, i.e. '*.bin.pb'"
+    ).file(exists = true, readable = true).multiple()
+    private val verbose by option("--verbose", "-v", help = "Print logs").flag(default = false)
+    // TODO(b/162273478) CLI should accept `policies` argument
+
+    private val fileBuilder by lazy {
+        FileSpec.builder(packageName, "")
+            .addComment("GENERATED CODE -- DO NOT EDIT")
+    }
+
+    /** Execute: Generate a plan per input manifest proto */
+    override fun run() = manifests
+        .also { DriverAndKeyConfigurator.configure(null) }
```

Comment:
Nit: function expressions should only be used when they fit on a single line, or when it opens a block on the first line.

Using a standard method body lets you list all the statements in the usual ordering, without having to contort things using `.also`.

So this then becomes:

```
overrride fun run() {
  DriverAndKeyConfigurator.configure(null)
  manifests.forEach { ... }
  outputFile.writeText(fileBuilder.build().toString())
}
```

which is, in my opinion at least, quite a bit clearer.

Also, FMI, what is this for: `DriverAndKeyConfigurator.configure(null)` ?

Reply:
Yeah, a normal block makes the most sense. Especially since I'm going to re-work things to process files one at a time. 
The configure call is needed in order to set up the driver providers so that storage key parsing works as expected.